forwardable/__init__.py

```diff
@@ -31,7 +31,7 @@ class NotCalledInModuleScope(Exception): pass
 class NotCalledInClassScope(Exception): pass
 class WrongDecoratorSyntax(Exception): pass
 
-def def_delegator(wrapped, attr_name, _call_stack_depth=1):
+def def_delegator(wrapped, attr_name, _call_stack_depth=1, obj_class=None):
```

Comment:
I'm not quite sure if `obj_class` is a good enough to inform user that "this will populate your __doc__ from this class".

Then again, naming things is hard and I can't think of a better name myself. Maybe I'll just go with `obj_class`.

Well, here is my sloppy candidate anyway: `doc_from_class`. What do you think?

Reply:
`doc_from_class` makes sense to me, I'll push an update

Reply:
Hey, I noticed that you haven't pushed an update yet. I think you must be busy handling the real life (I know I am).

Maybe I'll just merge this patch and update the argument name myself.

Also, I think I'll move the new argument's position from 4th to 3rd (i.e. put `doc_from_class` before `_call_stack_depth`) since `_call_stack_depth` is not part of public API. Better hide it way behind the argument list.

I hope it's okay with you since you might be the sole user of this new API.

Reply:
Hi yeah, sorry haven't gotten around to it. Feel free to merge and modify as you like.